faqs/galaxy/support_messaging_system.md

```diff
@@ -0,0 +1,116 @@
+
+* Getting help from the community
+* Discussing tool and workflow development
+* Organizing training events and materials
+* Collaborating on Galaxy infrastructure or scientific questions
+
+Matrix provides an open alternative to proprietary platforms like Slack and Microsoft Teams, with broader flexibility and user privacy in mind.
+
+---
+
+### 🔰 For Newbies: Getting Started with Matrix
+
+If you are new to messaging systems, here's how to get started with Galaxy on Matrix:
+
+<img src="{% link faqs/galaxy/images/support_matrix_element.jpg %}" alt="Galaxy Project Matrix space using the Element client app" width="350" align="right">
+
+#### Step 1: Choose and install a Matrix app
+
+**Install a Matrix client app:** Download and install a Matrix client:
+
+* [**Element**](https://element.io/) – Recommended. Available on [Web](https://app.element.io), [iOS](https://apps.apple.com/app/element/id1083446067), [Android](https://play.google.com/store/apps/details?id=im.vector.app), [Windows/macOS/Linux](https://element.io/download).&#x20;
+
+* [**Cinny**](https://cinny.in/)
+
+* [**Other Matrix clients...**](https://matrix.org/ecosystem/clients/)
+
+All of these apps connect to the same Galaxy channels/rooms. Choose the interface that works best for you.
+
+#### Step 2: Create a Matrix account
+
+* Open the app and sign up. It's easiest to use the default public server at `matrix.org` for your account.
+* Choose a username. Your Matrix ID will look like: `@yourname:matrix.org`.
+
+#### Step 3: Join the Galaxy **"Lobby"**
+
+* In your Matrix client, join the [main Galaxy chat room - The Lobby](https://matrix.to/#/#galaxyproject_Lobby:gitter.im).
+  * If you're using the Element client, click Explore (the compass icon) and search for "Galaxy" – look for a room named Galaxy or Galaxy Lobby (often with an address like `#galaxyproject:matrix.org`).
+  * Alternatively, you can use a direct Matrix link if provided, which will prompt your app to join the room.
+
+* In addition to the [Lobby room](https://matrix.to/#/#galaxyproject_Lobby:gitter.im), join other Matrix rooms via direct links once logged in:
+
+<div align="center">
+
+| Galaxy Subjects on Matrix:  | Galaxy Users on Matrix:  |
+|---|---|
+| [Proteomics](https://matrix.to/#/#GalaxyProteomics_Lobby:gitter.im) <br> [Microbiology](https://matrix.to/#/#galaxyproject_microGalaxy:gitter.im) <br> [Single-Cell & Spatial Omics Users](https://matrix.to/#/#Galaxy-Training-Network_galaxy-single-cell:gitter.im) <br> **[All Galaxy Matrix rooms, 70+ rooms](https://matrix.to/#/#galaxyproject:matrix.org)** | [Tool Authors](https://matrix.to/#/#galaxyproject_tools:gitter.im) <br> [Developers](https://matrix.to/#/#galaxyproject_dev:gitter.im) <br> [Outreach](https://matrix.to/#/#galaxyproject_wg-goat:gitter.im) <br> [Admins](https://matrix.to/#/#galaxyproject_admins:gitter.im) |
```

Comment:
This table isnt rendering nicely for me, I don't think it likes the multiline cells, so either make it a full html table, or just use single-line cells in markdown

Reply:
Thanks for the heads-up - updated.